TriggerBxMonitor: migrate to Stage 2 (80x)

fwyzard opened the pull request:
TriggerBxMonitor
- migrate to Stage 2 L1 Trigger
- correct the number of bx to 3564
- add 2D plots, remove plots for unused L1 triggers
- update test

fwyzard committed:
TriggerBxMonitor: migrate to Stage 2

TriggerBxMonitor
  - migrate to Stage 2 L1 Trigger
  - correct the number of bx to 3564
  - add 2D plots, remove plots for unused L1 triggers
  - update test

fwyzard commented:
please test

fwyzard commented:
+1

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
A new Pull Request was created by  @fwyzard (Andrea Bocci) for CMSSW_8_0_X.

It involves the following packages:

DQM/HLTEvF


@cmsbuild, @dmitrijus, @vanbesien, @deguio, @davidlange6 can you please review it and eventually sign? Thanks.
@slava77, @degano, @smuzaffar you are the release manager for this.

cms-bot commands are list here cms-sw/cmssw#13028

cmsbuild commented:
+1
Tested at: 82e09d790f500584e4274aac30d57ddda9e0bf78
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14338/12776/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14338/12776/summary.html

cmsbuild commented:
Pull request #14338 was updated. @cmsbuild, @dmitrijus, @vanbesien, @deguio, @davidlange6 can you please check and sign again.

fwyzard commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

fwyzard commented:
+1

fwyzard committed:
TriggerBxMonitor: monitor TCDS trigger types

TriggerBxMonitor
  - monitor the bx distribution for the TCDS trigger types
  - add Y axis labels to 2D plots

fwyzard committed:
DQM/HLTEvF/plugins/BuildFile.xml: remove invalid entry

cmsbuild commented:
Pull request #14338 was updated. @cmsbuild, @dmitrijus, @vanbesien, @deguio, @davidlange6 can you please check and sign again.

fwyzard commented:
please test

fwyzard commented:
+1

cmsbuild commented:
The tests are being triggered in jenkins.

fwyzard commented:
@vanbesien @dmitrijus could you sign this ?

dmitrijus commented:
+1
looks okay

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_8_0_X IBs after it passes the integration tests. This pull request requires discussion in the ORP meeting before it's merged. @slava77, @davidlange6, @degano, @smuzaffar

cmsbuild commented:
+1
Tested at: 28f7c2080dec639ac6cf2e07a98574e354cbd3be
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14338/12811/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14338/12811/summary.html

davidlange6 commented:
+1
